projects/components/src/button/button.component.scss

```diff
@@ -79,7 +79,7 @@
 
 .bordered {
   &.primary {
-    @include button-style(inherit, $blue-1, $gray-7, $gray-7, $blue-5);
+    @include button-style(inherit, $blue-1, $blue-5, $blue-4, $blue-5);
```

Comment:
nit: can you run lighthouse to see if this change supports accessibility?

Reply:
![Screenshot 2023-05-10 at 5 38 51 PM](https://github.com/hypertrace/hypertrace-ui/assets/52081890/cb0da08d-57ba-4b4b-82c7-7015c70828f4)
Looks good